### ISSUE TITLE
Remove key and value as required parameters

### ISSUE THREAD

**Issue by dyegos** [FutureWorkshops/bitrise-step-cocoapod-key#5]
When using this Bitrise's step we would like to omit values and keys parameters. 
Bitrise's Environment Keys will match our Podfile Cocoapods Keys. 
Making them not required will allow us to leverage this plugin to install Cocoapods keys instead of creating a custom script for it.

**Comment by igorcferreira** [comment]
@dyegos I've just pushed a fix for this under the tag 1.1.0. I'll open a PR to Bitrise to update it in their Step lib, but, if you want to quickly solve your issue for now, you can point to this Github project directly, similarly to:

```yml
- git::https://github.com/FutureWorkshops/bitrise-step-cocoapod-key@master
```

**Comment by igorcferreira** [comment]
@dyegos The latest version was merged into Bitrise's step lib, so I'm closing this issue. If you see any other problem, please let us know by creating a new issue. Happy building! :)
